src/server/project.ts

```diff
@@ -1827,11 +1831,16 @@ namespace ts.server {
             super.close();
         }
 
+        setTypeAcquisition(newTypeAcquisition: TypeAcquisition): void {
+            this.typeAcquisition = this.removeLocalTypingsFromTypeAcquisition(newTypeAcquisition);
+        }
+
         getTypeAcquisition(): TypeAcquisition {
             return {
                 enable: allRootFilesAreJsOrDts(this),
                 include: ts.emptyArray,
-                exclude: ts.emptyArray
+                exclude: ts.emptyArray,
+                inferTypings: this.typeAcquisition?.inferTypings

```

Comment:
This is incorrect..  if host,typeAcquisition exists thats the one you should be using and not just single setting. it gets confusing if host can set whole TypeAcquisition option but project uses only one

Reply:
Should I just set the entire `typeAcquisition` when creating an inferred object and just leave it alone after that like the other projects? Seems like we would need to call `allRootFilesAreJsOrDts` each time we add a new file anyway.

Reply:
this.typeAcquisition ||  {
                enable: allRootFilesAreJsOrDts(this),
                include: ts.emptyArray,
                exclude: ts.emptyArray
                exclude: ts.emptyArray }

So that if user has set the typeAq, then that takes preference over whatever we have set